pkg/util/uint128/uint128.go

```diff
@@ -30,6 +30,13 @@ func (u Uint128) GetBytes() []byte {
 	return buf
 }
 
+// AppendBytes appends big-endian byte representation to the
+// buffer and returns the buffer.
+func (u Uint128) AppendBytes(buf []byte) []byte {
```

Comment:
I don't see this function used anywhere. Am I missing something?

Reply:
We can leave this backport as-is and remove this function on master and backport that commit too. Or just remove it here if that's easier. I don't feel strongly.